internal/gatewayapi/listener.go

```diff
@@ -218,10 +218,13 @@ func processAccessLog(envoyproxy *configv1a1.EnvoyProxy) *ir.AccessLog {
 	return irAccessLog
 }
 
-func processTracing(envoyproxy *configv1a1.EnvoyProxy) *configv1a1.ProxyTracing {
+func processTracing(gw *v1beta1.Gateway, envoyproxy *configv1a1.EnvoyProxy) *ir.Tracing {
 	if envoyproxy == nil || envoyproxy.Spec.Telemetry.Tracing == nil {
 		return nil
 	}
 
-	return envoyproxy.Spec.Telemetry.Tracing
+	return &ir.Tracing{
+		ServiceName:  gw.Name + "." + gw.Namespace,
```

Comment:
lets use this helper func here instead ?
https://github.com/envoyproxy/gateway/blob/773bdfbefc769f51779d5056547e88d12e1d9508/internal/gatewayapi/helpers.go#L389

Reply:
sure, let me find a right package